src/main/java/org/apache/commons/imaging/formats/tiff/write/TiffOutputField.java

```diff
@@ -66,6 +66,10 @@ public boolean bytesEqual(final byte[] data) {
         return Arrays.equals(bytes, data);
     }
 
+    public boolean bytesEqual(final TiffOutputField other) {
+        return Arrays.equals(bytes, other.bytes);
+    }
```

Comment:
Reading the code on my mobile, but this was the only part that called my attention, everything else looks good (can look with the IDE later today/this week).

IMO we should see if we can move these methods to test class (or internal package), and these should not be exposed/shipped as public API for Imaging.

But the update looks fine. Just need some moments with ide and coverage tool to confirm.

Thanks!

Reply:
@kinow, appreciate your review!

I understand the importance of being cautious when introducing a new API. Given the existence of a closely related method above (primarily utilized by tests), I believed its inclusion would be acceptable.

I've implemented the requested change, and it's now package private.

Reply:
The method is still not package-private.

Reply:
@garydgregory It is. You are looking at the wrong commit.

These are the current lines:
```
    /* Only used in tests. */
    protected boolean bytesEqual(final TiffOutputField other) {
        return Arrays.equals(bytes, other.bytes);
    }
```